src/config/dynamic-theme-fixes.config

```diff
@@ -1065,6 +1065,19 @@ INVERT
 
 ================================
 
+aprs.fi
+*.aprs.fi
+
+INVERT
+.gm-style
+.gm-style-iw-a
+
+CSS
+#panel {
```

Comment:
![image](https://user-images.githubusercontent.com/25481501/124399855-dacb4500-dd0d-11eb-95f0-b01578df0274.png)
The panel looks good to me?

Reply:
If you take a closer look, "Login" href just above the cookies disclaimer is almost not visible with current background color.